docs/guides/hooks-list.rst

```diff
@@ -478,6 +478,12 @@ Embed
 
 **embed_get_upload_sections, all**
 
+Groups
+------
+
+**profile_buttons, group**
+	Filters buttons to be registered in the title menu of the group profile page
```

Comment:
Can convert to `ElggMenuItem`, doesn't make much difference. `elgg_register_menu_item()` that's called for each item returned accepts either array or `ElggMenuItem`